domain/src/main/assets/fractions_topic.json

```diff
@@ -1,7 +1,7 @@
 {
   "topic": {
     "story_reference_schema_version": 1,
-    "description": "",
+    "description": "You'll often need to talk about <b>part</b> of an object or group. For example, a jar of milk might be half-full, or some of the eggs in a box might have broken. In these lessons, you'll learn to use <b>fractions</b> to describe situations like these.",
```

Comment:
Can we also update the domain tests (i.e. topic controller) to verify that the description is passed properly?

Reply:
Sure, seems reasonable. Done.